src/measurements.jl

```diff
@@ -189,3 +33,37 @@ end
 function entanglemententropy(ρ0::LPDO; kwargs...)
   return error("Measurement of entanglement entropy for LPDOs not yet implemented")
 end
+
+function expect(T₀::ITensor, ops::AbstractString...; kwargs...)
```

Comment:
That's a great idea to have this. We should consider moving it to ITensors.jl but that is not urgent.

Reply:
I will add this together with the MPO version when ready